extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -225,10 +230,13 @@ public void build(ApplicationInfoBuildItem applicationInfo,
         //Apply configuration
         applyGlobalConfig(session, kubernetesConfig);
         ZonedDateTime now = ZonedDateTime.now(ZoneOffset.UTC);
+        ImagePullPolicy imagePullPolicy = determineImagePullPolicy(kubernetesConfig, containerImage, capabilities);
         applyConfig(session, project, KUBERNETES, getResourceName(kubernetesConfig, applicationInfo), kubernetesConfig,
```

Comment:
Should inline the `determineImagePullPolicy` call like the subsequent `applyConfig` calls do?

Reply:
Yeah, that makes sense.

Reply:
I must have missed it, thanks for pointing it out :)